### DISCUSSION TITLE
Documentation with GitHub pages

### DISCUSSION THREAD

**Discussion by drewejohnson** [CORE-GATECH-GROUP/serpent-tools#513] [Ideas]
@DanKotlyar do you want to switch over to GitHub pages from readthedocs? I think you had mentioned it in the past, just want to capture the discussion.

**Comment by DanKotlyar** [comment]
I can do that if you'd like?
In short, this only requires the inclusion of the GitHub pages branch.
Then, you can link the documentation from that branch using the WIKI

**Reply by drewejohnson** [reply]
Sounds good. If you want to take a crack at it, go for it. I'm not super familiar w/ github pages

**Comment by DanKotlyar** [comment]
I'll do that soon. I will also add the proper documentation and add to the developer's guide.
